pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -109,14 +109,14 @@ func (plugin *awsElasticBlockStorePlugin) NewCleaner(volName string, podUID type
 }
 
 func (plugin *awsElasticBlockStorePlugin) newCleanerInternal(volName string, podUID types.UID, manager pdManager, mounter mount.Interface) (volume.Cleaner, error) {
-	return &awsElasticBlockStore{
+	return &awsElasticBlockStoreCleaner{&awsElasticBlockStore{
 		podUID:      podUID,
 		volName:     volName,
 		manager:     manager,
 		mounter:     mounter,
 		diskMounter: &awsSafeFormatAndMount{mounter, exec.New()},
 		plugin:      plugin,
```

Comment:
I think we can remove unused attributes from the Cleaner and Builder.   For example, a Cleaner will never call diskMounter.  You would need to look at the functionality of the Cleaner to know which attributes can be removed from the cleaner struct.